ablySpec/Auth.swift

```diff
@@ -599,5 +599,164 @@ class Auth : QuickSpec {
                 expect(createTokenRequestMethodWasCalled).to(beTrue())
             }
         }
+
+        // RSA9
+        describe("createRequestToken") {
+
+            // RSA9a
+            it("should create and sign a TokenRequest") {
+                let options = AblyTests.commonAppSetup()
+                let rest = ARTRest(options: options)
+                let expectedClientId = "client_string"
+                let tokenParams = ARTAuthTokenParams(clientId: expectedClientId)
+
+                rest.auth.createTokenRequest(tokenParams, options: options, callback: { tokenRequest, error in
+                    expect(tokenRequest?.clientId).to(equal(expectedClientId))
```

Comment:
No check that it's actually a TokenRequest type, we should add that.  I appreciate it's a typed language, but then this code does not demonstrate that as the type is not specified.